### PR TITLE
Updating app_requested payload to include new field

### PR DESCRIPTION
**Pull request by gugutagli**
Updating ```app_requested``` payload to include ```is_granular_bot_app```

### Category (place an `x` in each of the `[ ]`)

* [x] **bolt** (Bolt for Java)
* [x] **bolt-{sub modules}** (Bolt for Java - optional modules)
* [x] **slack-api-client** (Slack API Clients)
* [x] **slack-api-model** (Slack API Data Models)
* [x] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [x] **slack-app-backend** (The primitive layer of Bolt for Java)

## Requirements

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you agree to those rules.

### CODE REVIEW

**Review by seratch — CHANGES_REQUESTED** [review]
This Java class needs to be updated and you can run the unit test that auto-generates the sample JSON file: https://github.com/slackapi/java-slack-sdk/blob/main/slack-api-model/src/main/java/com/slack/api/model/admin/AppRequest.java

**Comment by gugutagli** [comment]
> This Java class needs to be updated and you can run the unit test that auto-generates the sample JSON file: https://github.com/slackapi/java-slack-sdk/blob/main/slack-api-model/src/main/java/com/slack/api/model/admin/AppRequest.java

Thank you, I was able to add to the java file. This is my first time working on this, which test do I run and how do I run them?

File: slack-api-model/src/main/java/com/slack/api/model/admin/App.java [hunk 1]
```diff
@@ -12,6 +12,7 @@ public class App {
     private String privacyPolicyUrl;
     private String appHomepageUrl;
     private String appDirectoryUrl;
+    @SerializedName("is_granular_bot_app")
```
Comment (seratch) [inline slack-api-model/src/main/java/com/slack/api/model/admin/App.java:15]:
```suggestion
    @SerializedName("is_granular_bot_app")
    private boolean granularBotApp
```

**Review by seratch — CHANGES_REQUESTED** [review]

**Comment by seratch** [comment]
@gugutagli You can run [this script](https://github.com/slackapi/java-slack-sdk/blob/main/scripts/run_no_prep_tests.sh) for it, but it may take time. For this specific pull request, running only [this test](https://github.com/slackapi/java-slack-sdk/blob/main/slack-app-backend/src/test/java/test_locally/sample_json_generation/EventsApiPayloadDumpTest.java) should be fine.

File: slack-api-model/src/main/java/com/slack/api/model/admin/App.java (hunk 1 above)
Comment (seratch) [reply slack-api-model/src/main/java/com/slack/api/model/admin/App.java:15]: If this flag can be absent in some patterns, `private Boolean isGranularBotApp` without SerializedName annotation is suitable.

**Comment by seratch** [comment]
@gugutagli Thanks for sending this PR!

**Pull request merged**